code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -321,10 +321,7 @@
 					else
 						wound_flavor_text["[temp.name]"] = "<span class='warning'>[get_pronoun("He")] [get_pronoun("has")] [temp.get_wounds_desc()] on [get_pronoun("his")] [temp.name].</span><br>"
 				if(temp.status & ORGAN_BLEEDING)
-					if(thin_covering)
-						is_bleeding["[temp.body_part_class()]"] = "<span class='danger'>[get_pronoun("He")] [temp.covered_bleed_report(species.blood_type)]</span><br>"
-					else
-						is_bleeding["[temp.name]"] = "<span class='danger'>[get_pronoun("His")] [temp.name] is bleeding!</span><br>"
+					is_bleeding["[temp.name]"] = "<span class='danger'>[get_pronoun("His")] [temp.name] is bleeding!</span><br>"
```

Comment:
Use the appropriate SPAN macro instead of the raw html

Reply:
The what? I thought the strings were HTML? What do you mean the SPAN macro?

Reply:
They are, but we use macros to manage them where possible, so in this case you will want to use SPAN_DANGER

Reply:
So wait, how am I supposed to implement it? It's already using <span class='danger'>

Reply:
span class='danger'

Reply:
(also every other code snippet in the file uses some variant of span class=)

Reply:
```suggestion
					is_bleeding["[temp.name]"] = SPAN_DANGER("[get_pronoun("His")] [temp.name] is bleeding")+ "<br>"
```
Should work, give it a test to confirm

Reply:
What's the extra + " for?

Reply:
String concatenation with the newline, it could also be left in the span, should work either way